_episodes/04-performance.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Teaching as a Performance Art"
```

Comment:
After coaching faculty for 2 years in evidence-based teaching practices, I find that they don't need to be reminded teaching is like performing. Rather the opposite. I'm fine with keeping the "performing" language if we clarify that it is an *interactive* performance. What is the term for when the audience plays a critical role in shaping the act?